### PR TITLE
RNs-4.9.26 Added with Bug Fixes

### PR DESCRIPTION
**Pull request by Wiharris**
To be merged on **03-29**

Jira: https://issues.redhat.com/browse/OCPPLAN-8682

OCP version: Applicable only to **enterprise-4.9**

Direct Doc Preview Link: https://deploy-preview-43869--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-26

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-26"]
+=== RHBA-2022:0861 - {product-title} 4.9.26 bug fix and security update
+
+Issued: 2022-03-29
+
+{product-title} release 4.9.26 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1022[RHBA-2022:1022] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1021[RHSA-2022:1021] advisory.
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:2691]: These advisory numbers (1022) don't match the ones above, at the title of this section (0861).

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6843421[{product-title} 4.9.26 container image list]
+
+[id="ocp-4-9-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, TCP connections would not close after completing health checks in the Ingress Operator. Consequently, TCP connections would accumulate, causing a buildup in the `LoadBalancer`. This fix disables keepalives while establishing a connection. This results in the TCP connection closing after each health check, preventing buildup in the `LoadBalancer`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064586[*BZ#2064586*])
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:2700]: `keepalive`

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
...
+
+* Previously, low limit numbers for client throttling messages would appear while running clusters. Consequently, the increased number of custom resource definitions (CRDs) generated low numbers and restricted API discovery requests. This fix increases the limit number, resulting in messages appearing less frequently. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2045008[*BZ#2045008*])
+
+* Previously, the `oc debug` command did not allow users to set a connection timout. Consequently, users were never logged out of the cluster environment. This fix adds a time out feature to effectively close out of a cluster after a certain period. This results in clusters automatically closing after a certain period of inactivity to offset the `oc debug` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065302[*BZ#2065302*])
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:2704]: timeout

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc:2691]: Thanks for the catch! Advisory number updated!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Done! :)

File: release_notes/ocp-4-9-release-notes.adoc (hunk 3 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Done! :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 4]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-26"]
+=== RHBA-2022:1022 - {product-title} 4.9.26 bug fix and security update
+
+Issued: 2022-03-29
+
+{product-title} release 4.9.26 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1022[RHBA-2022:1022] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1021[RHSA-2022:1021] advisory.
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2691]:
```suggestion
{product-title} release 4.9.26, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1022[RHBA-2022:1022] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1021[RHSA-2022:1021] advisory.
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 5]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
...
+
+* Previously, low limit numbers for client throttling messages would appear while running clusters. Consequently, the increased number of custom resource definitions (CRDs) generated low numbers and restricted API discovery requests. This fix increases the limit number, resulting in messages appearing less frequently. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2045008[*BZ#2045008*])
+
+* Previously, the `oc debug` command did not allow users to set a connection timeout. Consequently, users were never logged out of the cluster environment. This fix adds a timeout feature to shut down a cluster after a period of inactivity. This results in clusters automatically closing after a prolonged period of idleness to offset the `oc debug` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065302[*BZ#2065302*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2704]:
```suggestion
* Previously, the `oc debug` command did not allow users to set a connection timeout, and users were never logged out of the cluster environment. With this update,  a timeout feature shuts down a cluster after a period of inactivity. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065302[*BZ#2065302*])
```
Just a suggestion for this one. I removed the last sentence bc it says pretty much the same thing as the sentence before. Take it or leave it. :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 6]
```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
...
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6843421[{product-title} 4.9.26 container image list]
+
+[id="ocp-4-9-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, TCP connections would not close after completing health checks in the Ingress Operator. Consequently, TCP connections would accumulate, causing a buildup in the `LoadBalancer`. This fix disables `keepalive` while establishing a connection. This results in the TCP connection closing after each health check, preventing buildup in the `LoadBalancer`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064586[*BZ#2064586*])
+
+* Previously, low limit numbers for client throttling messages would appear while running clusters. Consequently, the increased number of custom resource definitions (CRDs) generated low numbers and restricted API discovery requests. This fix increases the limit number, resulting in messages appearing less frequently. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2045008[*BZ#2045008*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2702]: I was looking through the ISG and I am unsure about throttling. `throttle` is a command. Not sure about the syntax here. 

Maybe: low limit numbers for client `throttle` messages. 

I am not sure though.

**Review by opayne1 — COMMENTED** [review]
Just a few comments/suggestions. :)

File: release_notes/ocp-4-9-release-notes.adoc (hunk 4 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Done! :)

File: release_notes/ocp-4-9-release-notes.adoc (hunk 6 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Yes, I believe it is a command as well!

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Pull request merged**